src/Features/CSharpTest/ConvertAutoPropertyToFullProperty/ConvertAutoPropertyToFullPropertyTests.cs

```diff
@@ -1384,14 +1404,44 @@ int [||]P
                     get => p;
                     set
                     {
-                        M(field);
-                        field = value;
+                        M(p);
+                        p = value;
```

Comment:
this was a bug fix.  when converting these to a full property, 'field' should be replaced with the final field name.